music_assistant/server/providers/deezer/__init__.py

```diff
@@ -526,21 +531,56 @@ def parse_playlist(self, playlist: deezer.Playlist) -> Playlist:
                     item_id=str(playlist.id),
                     provider_domain=self.domain,
                     provider_instance=self.instance_id,
+                    url=playlist.link,
                 )
             },
             metadata=MediaItemMetadata(
                 images=[MediaItemImage(type=ImageType.THUMB, path=playlist.picture_big)],
+                checksum=playlist.checksum,
             ),
-            is_editable=playlist.creator.id == self.client.user.id,
+            is_editable=creator.id == self.user.id,
+            owner=creator.name,
         )
 
-    def parse_track(
+    def get_playlist_creator(self, playlist: deezer.Playlist):
+        """See https://twitter.com/Un10cked/status/1682709413889540097."""
+        try:
+            return playlist.creator
+        except AttributeError:
+            return playlist.user
```

Comment:
>imho it's always better to check a condition instead of catching an exception, so I would change it as suggested

Yeah thats very true i didnt even think of that

Reply:
There is probably alot of other places we can change that as well